src/main/java/frc/robot/Constants.java

```diff
@@ -596,4 +596,39 @@ public static final class CraneConstants {
     public static final Translation2d kPositionIntake =
       new Translation2d(Units.degreesToRadians(32.042), 0.6824);
   }
+
+  public static final class ClimberConstants {
+    public static final boolean kEnable = false;
+    public static final int kMotorID = 0; //TODO: Get motor ID.
```

Comment:
We can assign an ID right now based on current configuration. 16 appears to be the next available CAN ID.